ru/lessons/advanced/otp-concurrency.md

```diff
@@ -8,7 +8,7 @@ lang: ru
 
 Мы уже рассматривали абстракции языка Elixir для параллельного выполнения, но иногда нужна более широкая функциональность и тогда мы обращаемся к поведениям OTP в языке.
 
-В этом уроке мы сосредоточимся на двух важных частях: GenServer и GenEvent.
+В этом уроке мы сосредоточимся на важнейшей части &mdash; GenServer.
```

Comment:
Doh!  Sorry @brain-geek, I should have wanted for you to review this first.  My bad 🙁